raiden/exceptions.py

```diff
@@ -305,3 +305,16 @@ class RawTransactionFailed(RaidenError):
 class UnhandledLightClient(RaidenRecoverableError):
     """Raised if someone tries to create a channel using this node as a hub and the light clients are not registered."""
 
+
+class ProxyTransactionError(RaidenError):
+    """ Raised when an operation is sent out to the blockchain and it returns an error and we need to handle it """
+
+    def __init__(self,
+                 tx_error_prefix: Optional[str],
+                 tx_error: Optional[Any],
+                 tx_gas_limit: Optional[int]):
+        super().__init__()
+        self.tx_error_prefix = tx_error_prefix
+        self.tx_error = tx_error
+        self.tx_gas_limit = tx_gas_limit
```

Comment:
- `super()` should be `super(RaidenError)` (it's _always_ better to specify which concrete parent class you are calling due to possible multiple inheritance)
- there is no need for the `tx_gas_limit` field, since this field is not read when the exception is raised.
- since the `RaidenError` parent class does not override the `__init__` method and it inherits from `Exception`, you should pass a `str` as an argument to the `super` call, as python exceptions should be initialized with an exception message.
  - `tx_error_prefix` is an acceptable argument here. it can also be something fancier than that, if you wish
- `tx_error_prefix` should not be optional, an exception without a message is unhelpful.

the rest looks good :+1:

Reply:
@JONAF2103 whoops, made a mistake in my code. check LGTM results (my bad!)

Reply:
Done